### PR TITLE
Add store_psql.py to store metrics in a postgresql db

### PR DESCRIPTION
**Pull request by lobstaj**
store_psql.py works very similarly to store_influxdb.py . It uses the psycopg2 package for connecting to the postgresql database.

Also updated README.md with the relevant information on the script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,4 +111,89 @@ set max electrical add.                   float
...
+
+Example:
+
+```bash
+cat examples/metrics.json | PSQL_USERNAME=myupway PSQL_PASSWORD=myupway python3 store_psql.py --host 10.110.1.6
+```
+
+Table schema of the `measurement` table:
+
+```
```
Comment (Jalle19) [inline README.md]: Could you add `sql` here for proper code formatting?

**Review by Jalle19 — COMMENTED** [review]
Very nice addition, thanks! The code looks good, if you add the README thing I'll merge this.

File: README.md (hunk 1 above)
Comment (lobstaj) [reply README.md]: Good point, should be in there now!

File: store_psql.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+"tehowatti addition blocked"                boolean,
+"inverter m8 compressor module blocked"     boolean,
+"test"                                      integer
+)
+'''
+
+# map duplicate measurement headers to separate columns
+column_mapping = {
+    "10033": "tehowatti addition blocked",
+    "10014": "inverter m8 compressor module blocked"
```
Comment (Jalle19) [inline store_psql.py:69]: Should we just change the column names in question so we don't need this mapping?

**Review by Jalle19 — APPROVED** [review]

**Pull request merged**